### ISSUE TITLE
Call for testing

### ISSUE THREAD

**Issue by ocefpaf** [conda-forge/pyferret-feedstock#5]
@emiliom and @PBrockmann I believe you may be interested in helping me test this recipe :wink: 
I am not a `ferret` user so I cannot do much beyond some simple plot tests I copied and pasted from the page.

You can install it with,

``` shell
conda create pyferret -n TEST_ENV -c conda-forge
source activate TEST_ENV
```

then call the CLI with `pyferret`.

Note that the `ipython magic` is also packaged here (also untested).

**Comment by emiliom** [comment]
@ocefpaf - gosh, I haven't used Ferret in a long time. When [I first expressed enthusiasm for pyferret](https://github.com/ioos/conda-recipes/issues/542), it wasn't b/c I'm currently using or plan to use Ferret in the near future. So I don't think I'm of much help. STILL, this is very cool!

**Comment by ocefpaf** [comment]
No problem. Based on the number of downloads I guess some people already using it. Let's hope they report back.

**Comment by karlmsmith** [comment]
Filipe --

I sent the info to the GFDL python users group email list this morning to
see if any were interested in trying it out.
Some feedback I got follows.  I would have assumed the netcdf-fortran would
have pulled in the netcdf, and the comment about netcdff (which is the
netcdf-fortran library) has me confused.  Is the problem that there are
multiple conda recipes with the same name but different sites?

And thank you very much for all your hard work on this.
I'm still trying to get the PyFerret documentation updated for v7.0.0
release, so my apologies for being rather absent from the conda effort.

Karl

On Tue, Jul 12, 2016 at 9:04 AM, Matthew Harrison - NOAA Federal <

> matthew.harrison@noaa.gov> wrote:
> I ran into the same issue. Solution was to build it myself and upload to
> my personal channel.  I am building my packages using conda libraries,
> _not_ using the system path.  We should avoid hacking conda to make it work
> at GFDL.
> 
> Karl, I suggest the PMEL team should take the lead on some of these common
> packages, including ESMF and NetCDF/HDF5 . There are other channels,
> including the defaults and IOOS which others here are using, but I find
> that they don't always work. I also needed libnetcdff , which is a bit
> harder to find.
> https://github.com/MJHarrison-GFDL/conda-recipes
> 
> > On Tue, Jul 12, 2016 at 9:01 AM, Seth Underwood - NOAA Federal <
> > seth.underwood@noaa.gov> wrote:
> > Setting a symlink to have libnetcdf.so.11 to the conda libnetcdf.so
> > works, at least for my test case.  Thus any user at GFDL can install this
> > as a conda package, as long as they manually create a simlink for the
> > missing libnetcdf.
> > 
> > On Tue, Jul 12, 2016 at 10:08 AM, Seth Underwood - NOAA Federal <
> > 
> > > seth.underwood@noaa.gov> wrote:
> > > Thank you for pointing us to this, however that conda package has
> > > pyferret linked against a shared netcdf library we do not have available on
> > > our systems (libnetcdf.so.11), and isn't part of any of the pyferret
> > > dependent packages.  I'll play around with creating symlinks to see if I
> > > can get it to work on our systems.

On Tue, Jul 12, 2016 at 3:27 AM, Filipe notifications@github.com wrote:

> No problem. Based on the number of downloads I guess some people already
> using it. Let's hope they report back.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> https://github.com/conda-forge/pyferret-feedstock/issues/5#issuecomment-232000430,
> or mute the thread
> https://github.com/notifications/unsubscribe/APpYx7NcoIZ0J8WaZMx1ESI3JnzGWaVnks5qU2wTgaJpZM4JIpPW
> .

## 

Karl M. Smith, Ph.D.
JISAO Univ. Wash. and PMEL NOAA
"The contents of this message are mine personally and do
not necessarily reflect any position of the Government
or the National Oceanic and Atmospheric Administration."

**Comment by ocefpaf** [comment]
> I sent the info to the GFDL python users group email list this morning to
> see if any were interested in trying it out.

Thanks!

> Some feedback I got follows.  I would have assumed the netcdf-fortran would
> have pulled in the netcdf, and the comment about netcdff (which is the
> netcdf-fortran library) has me confused.

`netcdf` is named `libnetcdf` in conda (long story, don't ask). I can confirm that `netcdf-fortran` does install `libnetcdf`.

> Is the problem that there are multiple conda recipes with the same name but different sites?

There are multiple sources for the netcdf family and multiple recipes, but as long as you have only the `defaults` and `conda-forge` channels in your `condarc` you should be good.

>  my apologies for being rather absent from the conda effort.

Don't worry. You answered all my questions as quick as I could act on them :smile: 

Regarding the `libnetcdf.so.11`, it does get installed. The only way for me to debug this is to get the them to do the issue the following commands:

``` shell
conda create pyferret -n PYFERRET -c conda-forge --yes
source activate PYFERRET
conda list  > debug.txt
conda info >> debug.txt
```

and send me the `debug.txt`file.

PS: There is one comment about building ESMF and NetCDF/HDF5 and the IOOS channel. Note that the IOOS channel is deprecated now (I was the maintainer there and we moved everything to `conda-forge` now) . And `conda-forge` has all those software and we welcomed people to use them and send feedback on how to do better builds.

See:
- https://github.com/conda-forge/hdf5-feedstock
- https://github.com/conda-forge/libnetcdf-feedstock
- https://github.com/conda-forge/esmf-feedstock

**Comment by PBrockmann** [comment]
Hi Folks,
Super work for the ease of install of PyFerret 7.0 from conda-forge. Thanks a lot.
I have tested the ferretmagic extension that you provide from the different notebook examples. They run perfectly. 
I have also updated the procedures of installation to get all this working together. It is visible from 
https://github.com/PBrockmann/ipython-ferretmagic and https://github.com/PBrockmann/ipython-ferretmagic/blob/master/ferretmagic_01_Discovering.ipynb
